src/ivalue_manager.rs

```diff
@@ -219,7 +220,7 @@ impl<'a> IValueKeyHolderWrite<'a> {
         mut op2_fun: F2,
     ) -> Result<Number, RedisError>
     where
-        F1: FnMut(i64, i64) -> i64,
+        F1: FnMut(i64, i64) -> (i64, bool),
```

Comment:
Whats the bool is used for?

Reply:
The bool returns whether there was an overflow.
Using `overflowing_add` and `overflowing_mul` returning the wrapped value if overflowed (should be the same result as with Release builds, only not skipping the overflow check)